VideoLocker/build.gradle

```diff
@@ -155,7 +155,7 @@ android {
     }
 
     compileSdkVersion 23
-    buildToolsVersion BUILD_TOOLS_VERSION
+    buildToolsVersion '23.0.3'
```

Comment:
This constant doesn't seem to satisfy the requirements for the rendscript error that comes up. http://developer.android.com/tools/revisions/build-tools.html so I explicitly set this.